tests/.nftignore

```diff
@@ -1,6 +1,12 @@
 .DS_Store
+Ancient_DNA/pydamage/analyze/*/*.csv
+Ancient_DNA/variant_calling/*/*.vcf.gz
+Annotation/Prokka/**/*.{log,err,gbk,sqn}
```

Comment:
No addition checks for gbk/sqn? These are common downstream files, so might be good to validate somehow